Sources/PayPalNativePayments/PayPalNativeShippingActions.swift

```diff
@@ -0,0 +1,25 @@
+import PayPalCheckout
+
+/// The actions that can be performed when the `PayPalNativeShippingDelegate` methods are invoked.
+public class PayPalNativeShippingActions {
```

Comment:
Nitpick - a rename suggestion to make it clear that `Actions` is meant for updating the Paysheet UI:

```suggestion
/// The actions that can be used to update the Paysheet UI after `PayPalNativeShippingDelegate` methods are invoked.
public class PayPalNativePaysheetActions {
```